src/trimfastq.py

```diff
@@ -0,0 +1,247 @@
+##################################
```

Comment:
I know this script is the kind of one that someone handed over, and it does a thing and it probably works and it is not the most fun thing to refactor/rewrite, but I think this should be done.  Especially the parts from line 108 forward where `i` gets assigned and reassigned would benefit from clarification. Parsing arguments with argparse would also clarify things.

Reply:
sure, I agree. I will ask Giorgi for it.